### PR TITLE
Added description of Provisioning improvements

### PR DESCRIPTION
**Pull request by ianf77**
Icon for supported templates
Filtering of templates.
Cloning supported templates

[doc] - Provisioning Improvements

https://issues.redhat.com/browse/SAT-6406

Cherry-pick into:

* [ X] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Comment by ianf77** [comment]
Not quite sure why it's reporting errors in the Upgrading guide.

**Comment by ianf77** [comment]
Could someone look at the error here please?

**Comment by lzap** [comment]
Builds fine locally, please rebase on top of `master`.

**Comment by lzap** [comment]
@ares can you confirm? This is the new feature I haven't seen yet, can't comment on that.

**Comment by ianf77** [comment]
Rebased

**Comment by ianf77** [comment]
@lzap @ares I've rebased on top of master as requested. It still shows an error here. Can this be reviewed and or merged please?

**Comment by ekohl** [comment]
It's not really rebased on master. Current master is 28f70052552b106545f33b69625931fc516f2051 while you're basing it on bf22029e7ef65066cbeb464baf90215ad2816a06. In between those are some crucial build fixes. Perhaps you forgot to update the branch you're rebasing on?

**Comment by ianf77** [comment]
I thought I'd done it (with help) correctly. I'm afraid rebasing is a bit of a dark art.

**Comment by ianf77** [comment]
@ekohl I think this is rebased successfully now.

File: guides/common/modules/con_provisioning-templates.adoc [hunk 1]
```diff
@@ -8,6 +8,15 @@ In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning templates* to view th
 You can create a template or clone a template and edit the clone.
 For help with templates, navigate to *Hosts* > *Provisioning templates* > *Create Template* > *Help*.
 
+ifdef::satellite[]
+Templates supported by {Team} are indicated by a {Team} icon.
+
+To show or hide unsupported templates, in the {ProjectWebUI} navigate to *Admininster > Settings*.
```
Comment (Lennonka) [inline guides/common/modules/con_provisioning-templates.adoc:14]: Administer :)

File: guides/common/modules/con_provisioning-templates.adoc [hunk 2]
```diff
@@ -8,6 +8,15 @@ In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning templates* to view th
...
+On the *Provisioning* page, change the value of *Show unsupported provisioning templates* and click *Submit*.
```
Comment (Lennonka) [inline guides/common/modules/con_provisioning-templates.adoc:15]: Provisioning *tab* perhaps?

Comment (Lennonka) [reply guides/common/modules/con_provisioning-templates.adoc:15]: ..., *set* the value of \*Show unsupported...\* to \`true\` and click \*Submit\*.

**Review by Lennonka — CHANGES_REQUESTED** [review]
Caught a typo and suggested rephrasing a bit.

**Review by lzap — APPROVED** [review]
Thanks

**Comment by lzap** [comment]
@Lennonka please re-review then this can be merged, no technical issues thanks.

**Review by Lennonka — APPROVED** [review]
LGTM

**Review by maximiliankolb — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Cherry-picked to 3.1: https://github.com/theforeman/foreman-documentation/commit/4c6af4bbbfe310375b4e1321279fb6f9fc2715a1 ✅

**Pull request merged**